src/main/public/stylesheets/components/_form.scss

```diff
@@ -3,6 +3,9 @@
 //noinspection CssUnknownTarget this file is generated by gulp
 @import 'colours';
 
+.title, #title {
```

Comment:
I don't see where title is being used as a class any where?

Reply:
we have a input field title in your details and the id of the input is title

Reply:
that will be covered by the #title we don't need the .title that I can see?

Reply:
agree , just added for future any title / or similar field which doesn't have a id title, I will remove it for now